src/base/bittorrent/session.cpp

```diff
@@ -384,12 +384,12 @@ Session::Session(QObject *parent)
     , m_coalesceReadWriteEnabled(BITTORRENT_SESSION_KEY("CoalesceReadWrite"), false)
 #endif
     , m_isSuggestMode(BITTORRENT_SESSION_KEY("SuggestMode"), false)
-    , m_sendBufferWatermark(BITTORRENT_SESSION_KEY("SendBufferWatermark"), 500)
-    , m_sendBufferLowWatermark(BITTORRENT_SESSION_KEY("SendBufferLowWatermark"), 10)
-    , m_sendBufferWatermarkFactor(BITTORRENT_SESSION_KEY("SendBufferWatermarkFactor"), 50)
+    , m_sendBufferWatermark(BITTORRENT_SESSION_KEY("SendBufferWatermark"), 16384)
+    , m_sendBufferLowWatermark(BITTORRENT_SESSION_KEY("SendBufferLowWatermark"), 1600)
```

Comment:
... you do realize the two numbers above will be multiplied by 1024 when passed to libtorrent?

Reply:
Mistake, will fix

Reply:
>Too high?

I don't know, since you're tweaking it, you should give reasons to justify the change.

Reply:
> > Too high?
> 
> I don't know, since you're tweaking it, you should give reasons to justify the change.

See my comment above: https://github.com/qbittorrent/qBittorrent/pull/11637#issuecomment-564712175

These values have been working fine for me even in machines with only 2 GiB RAM, though in my use case it is fine if qbit consumes 700+ MiBs which is probably not ok in the general case, and I don't know how much of that memory usage is due to these settings. I am certain that increasing the values from the current defaults is the right thing to do, I am just not sure by _how much_  they should be increased. Even in my use cases, I might already be in snake-oil territory, but I can't be sure. That's why I need help figuring out a way to come up with reasonable values.